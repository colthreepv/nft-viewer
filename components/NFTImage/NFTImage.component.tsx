import { FC } from 'react'
import Image from 'next/image'
import styled from 'styled-components'

interface NFTImageProps {
  collectionName: string
  tokenId: string
  src: string
}

const HoverImage = styled(Image)`
  transition: box-shadow 0.1s linear;
  &:hover {
    box-shadow: 0px 0px 20px 0px rgba(255, 255, 255, 0.5);
  }
`

const NFTImage: FC<NFTImageProps> = ({ collectionName, tokenId, src }) => {
  return (
    <div className="relative min-h-[50vh] md:min-h-[33vh] cursor-zoom-in">
      <HoverImage className="object-cover" src={src} alt={`${collectionName} #${tokenId}`} fill />
    </div>
  )
}

export default NFTImage
