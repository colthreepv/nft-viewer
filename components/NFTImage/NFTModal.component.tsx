/* eslint-disable @next/next/no-img-element */
import { NFTImageProps } from './NFTImage.component'
import { Transition, Dialog } from '@headlessui/react'
import { FC, Fragment } from 'react'
import { useModalContext } from '@/hooks/use-modal.context'

const NFTModal: FC<NFTImageProps> = ({ collectionName, tokenId, src }) => {
  const { isOpen, closeModal } = useModalContext()

  return (
    <Transition appear show={isOpen} as={Fragment}>
      <Dialog as="div" className="relative z-10" onClose={closeModal}>
        <Transition.Child
          as={Fragment}
          enter="ease-out duration-300"
          enterFrom="opacity-0"
          enterTo="opacity-100"
          leave="ease-in duration-200"
          leaveFrom="opacity-100"
          leaveTo="opacity-0"
        >
          <div className="fixed inset-0 bg-black/90" />
        </Transition.Child>

        <div className="fixed inset-0 overflow-y-auto">
          <div className="min-h-full min-w-full p-6 flex items-center justify-center">
            <Transition.Child
              as={Fragment}
              enter="ease-out duration-300"
              enterFrom="opacity-0 scale-95"
              enterTo="opacity-100 scale-100"
              leave="ease-in duration-200"
              leaveFrom="opacity-100 scale-100"
              leaveTo="opacity-0 scale-95"
            >
              <Dialog.Panel className="rounded-md overflow-hidden cursor-zoom-out">
                <img src={src} alt={`${collectionName} #${tokenId}`} onClick={closeModal} />
              </Dialog.Panel>
            </Transition.Child>
          </div>
        </div>
      </Dialog>
    </Transition>
  )
}

export default NFTModal
