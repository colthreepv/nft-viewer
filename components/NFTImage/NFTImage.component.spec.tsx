import { render } from '@testing-library/react'
import NFTImage, { NFTImageProps } from './NFTImage.component'

describe('NFTImage', () => {
  it('Matches snapshot', async () => {
    const props: NFTImageProps = {
      collectionName: 'SayCheese By KimiNaki',
      tokenId: '1',
      src: 'https://imgproxy-mainnet.avascan.com/mnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY/thumb_1024/czM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw',
    }

    const rendered = render(<NFTImage {...props} />)

    const { findByAltText } = rendered

    const element = await findByAltText(`${props.collectionName} #${props.tokenId}`)
    expect(rendered).toMatchSnapshot()
    expect(element).toBeInTheDocument()
  })
})
