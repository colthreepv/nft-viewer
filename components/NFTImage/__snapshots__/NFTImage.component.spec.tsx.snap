// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NFTImage Matches snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="relative min-h-[50vh] md:min-h-[33vh] cursor-zoom-in"
      >
        <img
          alt="SayCheese By KimiNaki #1"
          class="sc-bcXHqe wyUiv object-cover"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="relative min-h-[50vh] md:min-h-[33vh] cursor-zoom-in"
    >
      <img
        alt="SayCheese By KimiNaki #1"
        class="sc-bcXHqe wyUiv object-cover"
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        sizes="100vw"
        src="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75"
        srcset="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`NFTImage Renders fixed image 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="relative min-h-[50vh] md:min-h-[33vh] cursor-zoom-in"
      >
        <img
          alt="SayCheese By KimiNaki #1"
          class="sc-bcXHqe wyUiv object-cover"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="relative min-h-[50vh] md:min-h-[33vh] cursor-zoom-in"
    >
      <img
        alt="SayCheese By KimiNaki #1"
        class="sc-bcXHqe wyUiv object-cover"
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        sizes="100vw"
        src="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75"
        srcset="/_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fimgproxy-mainnet.avascan.com%2FmnxeMftCs4-mwNwc9jyj65h50U47fsxVu5qa9ri4jyY%2Fthumb_1024%2FczM6Ly9hdmFzY2FuLW1haW5uZXQtbmZ0L2VyYzcyMS80MzExNC9kNi9mNy9iMi9lMy9kMmJlNWMxMjAzOGIyNGFmY2Q3NzllMmYxNzliMjQwMC8xLnBuZw&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
